hitboxYStickDeDosEjes.js

```diff
@@ -20,17 +20,17 @@
  */
 //valores de hardware:
 //mano derecha: //right hand:
-// posicion del boton en la hitbox = codigo del boton en esa posicion dado por https://html5gamepad.com
```

Comment:
editing the header to fit the new controller scheme will be nice.
Not a big deal tho